src/main/webapp/app/quiz/participate/multiple-choice-question/multiple-choice-question.html

```diff
@@ -0,0 +1,45 @@
+<div class="mc-question">
+    <h2>{{vm.question.title}}</h2>
+    <p ng-bind-html="vm.rendered.text">
+    </p>
+    <h4>
+        <span
+            class="label label-info"
+            uib-popover-html="vm.rendered.hint"
+            popover-placement="right-top"
+            popover-trigger="mouseenter"
+            ng-if="vm.question.hint"
+        >
```

Comment:
Same comment about code formatting as in the other HTML file. I prefer writing tags onto one line, except there is a good reason not to do it